time stamp
int64 id
geometry_msgs/Pose pose
int64 number_tracked_features
